### PR TITLE
Fixing issue https://github.com/openshift/openshift-docs/issues/25135

### PR DESCRIPTION
**Pull request by rounaksarda**
Issue raise by @hupiper 

Fixing the below issue :
https://github.com/openshift/openshift-docs/issues/25135

an changed to and.

### CODE REVIEW

**Review by rounaksarda — COMMENTED** [review]
fixing the issue:
https://github.com/openshift/openshift-docs/issues/25814

Earlier the sentence was "In this document, this document" -> "In this, document might"

**Comment by vikram-redhat** [comment]
@rounaksarda thanks! I will accept this for review by the team. In the future, please open a separate PR for each issue.

**Comment by vikram-redhat** [comment]
@openshift/team-documentation PTAL.

File: modules/cluster-logging-about-components.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ The major components of cluster logging are:
 * visualization - This is the UI component you can use to view logs, graphs, charts, and so forth. The current implementation is Kibana.
 
 ifndef::virt-logging[]
-In this document, this document might refer to log store or Elasticsearch, visualization or Kibana, collection or Fluentd, interchangeably, except where noted.
+In this, document might refer to log store or Elasticsearch, visualization or Kibana, collection or Fluentd, interchangeably, except where noted.
```
Comment (bergerhoffer) [inline modules/cluster-logging-about-components.adoc:25]: This still doesn't read very well. Suggest changing it to say simply "This document might..."

**Review by bergerhoffer — COMMENTED** [review]
@rounaksarda Thanks for submitting these fixes!

I commented with one update that needs to be made.

And then before this can be merged, you'll need to squash your commits so that the PR only has 1 commit. Let me know if you need any help with how to do this. Thanks!

**Review by rounaksarda — COMMENTED** [review]
@bergerhoffer I have changed as you have suggested, let me know if something is still not quite expected.

File: modules/cluster-logging-about-components.adoc (hunk 1 above)
Comment (rounaksarda) [reply modules/cluster-logging-about-components.adoc]: > @rounaksarda thanks! I will accept this for review by the team. In the future, please open a separate PR for each issue.

@vikram-redhat  ok, I will take care next time. Thanks.

Comment (rounaksarda) [reply modules/cluster-logging-about-components.adoc]: @bergerhoffer ok, suggested change is made

**Comment by bergerhoffer** [comment]
Hi @rounaksarda we still need you to squash your commits. Right now there are 3 (you can see them listed in the "Commits" tab), but there should only be 1.

There are some instructions here: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request#squash-your-changes

But feel free to reach out if you need help on how to do it. Thanks!

**Comment by rounaksarda** [comment]
Hello @bergerhoffer , I have gone through the link that you shared but in my case things are quite complicated . 
First I forked this repo and directly added the changes to the file(without cloning or making other branch) and raised the pull request .
second I again made some changes and raise a pull request .
third time I made a branch new and added the change there and raised a pull request.

I can settle this in easy way by deleting all of this and follow the the fork then clone and then make a branch and commit all together and raise a single pull request.

OR 

Please you guide me how can I make it right. I have gone through the link that you shared but was not able to set up.

**Comment by bergerhoffer** [comment]
> Hello @bergerhoffer , I have gone through the link that you shared but in my case things are quite complicated .
> First I forked this repo and directly added the changes to the file(without cloning or making other branch) and raised the pull request .
> second I again made some changes and raise a pull request .
> third time I made a branch new and added the change there and raised a pull request.
> 
> I can settle this in easy way by deleting all of this and follow the the fork then clone and then make a branch and commit all together and raise a single pull request.
> 
> OR
> 
> Please you guide me how can I make it right. I have gone through the link that you shared but was not able to set up.

So if it's easier for you to just create a new branch to make the updates, that's totally fine. I just don't want you to miss your Hacktoberfest acknowledgement if that's what you were working toward.

But if you'd like to try to squash the commits, I've outlined the steps of how to do so below:

1. In the branch in your terminal, use this command to squash your 3 commits into 1:
```
$  git rebase -i HEAD~3
```

2. In the file that pops up, change the "pick" to "squash" for all but the first entry, like below:
```
pick 8f95fae21 Fixing issue https://github.com/openshift/openshift-docs/issues/25135
squash 58ed8b8df Update cluster-logging-about-components.adoc
squash 73d28bbe4 Update cluster-logging-about-components.adoc
```
3. Save and quit the file (if it's vim, enter `:wq`).

4. In the next file that pops up, you can remove the extra 2 commit messages, like below

```
# This is a combination of 3 commits.
# This is the 1st commit message:

Fixing issue https://github.com/openshift/openshift-docs/issues/25135

# This is the commit message #1:

Update cluster-logging-about-components.adoc                                 <---- Remove this line
# This is the commit message #2:

Update cluster-logging-about-components.adoc                                 <---- Remove this line
```

5. Save and quit the file (if it's vim, enter `:wq`).

6. Force push the changes to this PR.

```
$ git push -f origin HEAD
```

When you look at this PR again, you should only see 1 commit in the "Commits" tab.

Let me know if that works for you, or if you have any issues in squashing. Thanks!

**Comment by rounaksarda** [comment]
Hello, @bergerhoffer I am not here looking for hacktoberfest, I want to try this squash method and followed all steps that you send. It results, the commit become 5. Any comments are welcome.

**Comment by bergerhoffer** [comment]
Okay great. I'm not sure how you ended up with 5 commits, but let's try another method for squashing. We'll basically undo your 5 commits (but not undo your changes), and reset to the commit before yours, and then have you commit again.

In the terminal in your branch:

1. Do a **soft** reset to the commit before your first. Looking at git log, it is commit ID `05a9ec2460f0a3ab1bb1010786b1da3e9d826cbf`:

```
$ git reset --soft 05a9ec2460f0a3ab1bb1010786b1da3e9d826cbf
```

2. Commit your changes:

```
$ git commit -m "Fixing issue #25135"
```

3. Force push the changes to this PR.

```
$ git push -f origin HEAD
```

If this doesn't end up working for you, let me know if you'd like me to try to squash for you in this PR. 

Thanks again for working on this!

**Comment by rounaksarda** [comment]
Hey, It worked @bergerhoffer, thanks. it is wonderful learning and fun contribution actually. 
The problem I got 5 commits earlier because I opened the original downloaded repo not one that I forked, while committing this time I checked the commit log and there was someone else's commit too. So I basically got the idea.... It was silly mistake by me.

Thank you!!

**Comment by bergerhoffer** [comment]
Great, I'm glad that worked for you! This looks great now, thanks for sticking with us to get this completed. I'm glad you enjoyed it, and we appreciate the contribution!

Merging now :)

**Comment by bergerhoffer** [comment]
Closing as the related PR has been merged!

**Pull request merged**